indico/modules/events/abstracts/controllers/abstract_list.py

```diff
@@ -107,12 +107,20 @@ def _process(self):
 
 class RHCreateAbstract(RHAbstractListBase):
     def _process(self):
-        abstract_form_class = make_abstract_form(self.event, session.user, notification_option=True, management=self.management)
+        is_invited = request.args.get('invited') == '1'
+        abstract_form_class = make_abstract_form(self.event, session.user, notification_option=True,
+                                                 management=self.management, invited=is_invited)
         form = abstract_form_class(event=self.event, management=self.management)
+        if is_invited:
+            to_remove = ['description', 'submitted_contrib_type', 'attachments', 'send_notifications', 'person_links']
```

Comment:
Isn't this a bit overengineered? ;)
Using separate `del` statements is only two more lines, and I think it's nicer than having one very long line with all the field names

```python
del form.description
del form.submitted_contrib_type
del form.attachments
del form.send_notifications
del form.person_links
```

Anyway, matter of personal preference..